code/game/machinery/cloning.dm

```diff
@@ -302,13 +315,14 @@
 	attempting = 0
 	return 1
 
-//Grow clones to maturity then kick them out.  FREELOADERS
+//Grow clones to maturity then kick them out. FREELOADERS
 /obj/machinery/clonepod/process()
 	var/show_message = 0
-	for(var/obj/item/reagent_containers/food/snacks/meat/meat in range(1, src))
-		qdel(meat)
-		biomass += BIOMASS_MEAT_AMOUNT
-		show_message = 1
+	for(var/obj/item/item in range(1, src))
+		if(is_type_in_list(item, GLOB.cloner_biomass_items))
+			qdel(item)
+			biomass += BIOMASS_BASE_AMOUNT
+			show_message = 1
```

Comment:
show_message is a boolean value. Standard is to use the defines `TRUE` or `FALSE` for those.
It's also set to 0 above on line 320. That one can also be replaced by a bool define

Reply:
I just worked my code around what was there previously, didn't look closely enough to realize that it could be replaced with the defines. Will do!